### ISSUE TITLE
Partial Save warnings under Settings -> Mail/Reporting/DNS

### ISSUE THREAD

**Issue by netniV** [Cacti/cacti#1443]
When saving this form, I repeatedly got a message saving the save failed, but no fields were highlighted.  No errors appeared in the cacti.log but the data was saved.

**Comment by cigamit** [comment]
It's the modal nature of the save.  Can you share a screen shot when it's happening?

**Comment by netniV** [comment]
It's just the standard failed message and no fields have the red outline.  But I'll try and get one done later.

**Comment by Herald1989** [comment]
I've experienced this as well. There is a highlighted box but it only appears very abruptly and then disappears. I was able to screencap it. It only pops up for a fraction of a second. Normally when you select either PHP Mail or SMTP, the sendmail path is no longer available, but when saving these options it makes its appearance momentarily and throws the error. I can confirm that the settings still save regardless of this error.

Cacti: 1.1.36
Linux: Ubuntu Server 16.04LTS

![cacti](https://user-images.githubusercontent.com/37186297/37160216-b3e6802a-22bd-11e8-93b1-2b9df4c68d22.JPG)
